packages/bpk-scrim-utils/src/scroll-utils.tsx

```diff
@@ -31,7 +31,7 @@ const getScrollBarWidth = () => {
   const window = getWindow();
   const body = getBodyElement();
 
-  if (body === null && window === null) {
+  if (body === null || window === null) {
```

Comment:
Could you confirm why we have needed to make this change here to the logic?
As this seems a significant change to the statement which could impact consumers/usage of the scrim utils